examples/00_tutorials/plot_single_subject_single_run.py

```diff
@@ -270,7 +270,7 @@
 # We can furthermore extract and report the found positions in a table.
 
 from nilearn.reporting import get_clusters_table
-table = get_clusters_table(z_map, stat_threshold=threshold,
+table, cluster_maps = get_clusters_table(z_map, stat_threshold=threshold,
```

Comment:
since you don't use the value

```suggestion
table, _ = get_clusters_table(z_map, stat_threshold=threshold,
```